src/components/ModListUpdateBanner.vue

```diff
@@ -0,0 +1,78 @@
+<script lang="ts">
+import { mixins } from 'vue-class-component';
+import { Component, Watch } from 'vue-property-decorator';
+
+import UtilityMixin from './mixins/UtilityMixin.vue';
+import ManagerInformation from '../_managerinf/ManagerInformation';
+
+@Component({})
```

Comment:
Minor nit: I don't believe we need the `{}`

Reply:
One argument (options) is required.